src/blocks.js

```diff
@@ -23,6 +23,7 @@ import './extensions/button-controls';
 import './extensions/image-styles';
 import './extensions/cover-styles';
 import './extensions/image-crop';
+import './extensions/coblocks-options/';
```

Comment:
Let's rename this directory and subsequent classes/files to `coblocks-settings` for consistency (coblocks-options-modal.js).

Reply:
Renamed